pkg/transport/socket.go

```diff
@@ -169,11 +167,11 @@ func (s *socketTransport) EnableCompression(level compress.Level) {
 }
 
 func (s *socketTransport) SupportsChannelBinding() bool {
-	return s.supportsCb
+	return true
```

Comment:
Not sure what's the intention here. Does this mean we are dropping support for all non `*-PLUS` auth mechanisms here? 🤔

Reply:
Oh, now I start to remember... seems that socket CB support was disabled for TLS >= 1.3. 

If this is going to be the case, shouldn't we simply get rid of the `SupportsChannelBinding` method? Given that it's going to be always enabled.

Reply:
We could probably just remove this method; I'm not sure why I left it in. Removed.

Reply:
Update: see main conversation. This is required for the `Transport` interface.